test/fixtures/fingerprinting-blocking-from-child-frames.html

```diff
@@ -0,0 +1,28 @@
+<html>
+<head>
+</head>
+<body>
+<div id="target">
+proxy blocking being tested
+</div>
+<script>
+(function () {
+
+    let parentFrameMethod = window.HTMLCanvasElement.prototype.toDataURL
+
+    let iframe = document.createElement("iframe")
+    iframe.src = "https://www.brave.com/"
```

Comment:
i think this fails now because brave.com sets the CSP frame-ancestors directive to prevent being iframed. you can use https://example.com instead.